examples/preprocessing/plot_css.py

```diff
@@ -0,0 +1,70 @@
+"""
+=================================================================
+Use Cortical Singal Suppression (CSS) to remove cortical signals
+=================================================================
+
+This script shows an easy example of how to use CSS. CSS
+suppresses the cortical contribution to the signal subspace
+in magnetometer or EEG data, facilitating detection of subcortical
+signals. We will illustrate how it works by simulating one cortical
+and one subcortical oscillation at different frequencies; 40 Hz and
+239 Hz for cortical and subcortical activity, respectively, then
+process it with CSS and look at the power spectral density of the
+raw and processed data.
+
+Presented in Samuelsson J, Khan S, Sundaram P, Peled N, Hamalainen M
+(2018) Cortical Signal Suppression (CSS) for detection of subcortical activity
+using MEG and EEG, Brain topography 32 (2), 215-228.
+Please refer to this paper.
```

Comment:
we have a standard way to format references. See for example:

https://github.com/mne-tools/mne-python/blob/master/examples/inverse/plot_mixed_norm_inverse.py#L11